assets/js/app.js

```diff
@@ -0,0 +1,14 @@
+// A User should see a start screen
+$('.game').html(AppTemplates.start(heroes));
+
+$('.start-name').click(function() {
```

Comment:
Change this to `$('.start-name').on('click', function() {`... The `click` function doesn't quite behave the same way that you would expect and can become confusing.